Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2015,6 +2086,8 @@ - (WKNavigation *)loadFileRequest:(NSURLRequest *)request allowingReadAccessToUR
     isAppInitiated = request.attribution == NSURLRequestAttributionDeveloper;
 #endif
 
+    [self _initializeScreenTimeWebpageController];
```

Comment:
Does this really need to be sprinkled in every load method?

Reply:
Yeah, it _might_ be cleaner to add a method to `PageClient`, that's called from inside `WebPageProxy::prepareToLoadWebPage`. The page client method could then call  `-[WKWebView _initializeScreenTimeWebpageController]` — that way, all ways of navigating web views would be covered.